pub/scripts/Utils.js

```diff
@@ -118,9 +118,11 @@ Utils.getTrimmedExtent = function (array) {
   var lastNonEmptyItem = 0
 
   array.forEach(function (value, i) {
-    if (value === '' && i === firstNonEmptyItem) {
+    var isEmpty = value === '' || value === null || value.toLowerCase() === 'null'
```

Comment:
You can use `value` here otherwise: `if (value && i === firstNonEmptyItem) { ... }`